From 189cfdbb9e30210ae58509bd73f7373f475ebaad Mon Sep 17 00:00:00 2001
From: Scott Talbert <swt@techie.net>
Date: Wed, 15 Nov 2017 19:37:52 -0500
Subject: [PATCH] Fix EVPTest by removing hard-coded EC curve name

---
 Crypto/testsuite/src/EVPTest.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Crypto/testsuite/src/EVPTest.cpp b/Crypto/testsuite/src/EVPTest.cpp
index 4a95fedd0..9ae91ab1a 100644
--- a/Crypto/testsuite/src/EVPTest.cpp
+++ b/Crypto/testsuite/src/EVPTest.cpp
@@ -336,7 +336,7 @@ void EVPTest::testECEVPSaveLoadStream()
 			assert (strPubE.str() == pubKey);
 			assert (key == key2);
 			assert (!(key != key2));
-			ECKey ecKeyNE("secp112r2");
+			ECKey ecKeyNE(curveName);
 			EVPPKey keyNE(&ecKeyNE);
 			assert (key != keyNE);
 			assert (!(key == keyNE));
@@ -391,7 +391,7 @@ void EVPTest::testECEVPSaveLoadStreamNoPass()
 			assert (strPubE.str() == pubKey);
 			assert (key == key2);
 			assert (!(key != key2));
-			ECKey ecKeyNE("secp112r2");
+			ECKey ecKeyNE(curveName);
 			EVPPKey keyNE(&ecKeyNE);
 			assert (key != keyNE);
 			assert (!(key == keyNE));
@@ -448,7 +448,7 @@ void EVPTest::testECEVPSaveLoadFile()
 			assert (strPubE.str() == pubKey);
 			assert (key == key2);
 			assert (!(key != key2));
-			ECKey ecKeyNE("secp112r2");
+			ECKey ecKeyNE(curveName);
 			EVPPKey keyNE(&ecKeyNE);
 			assert (key != keyNE);
 			assert (!(key == keyNE));
-- 
2.14.3

