From 0652263078c78d3914e75496922aab51e3c0749a Mon Sep 17 00:00:00 2001
From: Scott Talbert <swt@techie.net>
Date: Tue, 14 Nov 2017 00:32:05 -0500
Subject: [PATCH] Fix EVPTest by switching to different EC Curve

---
 Crypto/testsuite/src/EVPTest.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Crypto/testsuite/src/EVPTest.cpp b/Crypto/testsuite/src/EVPTest.cpp
index 4a95fedd0..0440d3fcd 100644
--- a/Crypto/testsuite/src/EVPTest.cpp
+++ b/Crypto/testsuite/src/EVPTest.cpp
@@ -336,7 +336,7 @@ void EVPTest::testECEVPSaveLoadStream()
 			assert (strPubE.str() == pubKey);
 			assert (key == key2);
 			assert (!(key != key2));
-			ECKey ecKeyNE("secp112r2");
+			ECKey ecKeyNE("secp256k1");
 			EVPPKey keyNE(&ecKeyNE);
 			assert (key != keyNE);
 			assert (!(key == keyNE));
@@ -391,7 +391,7 @@ void EVPTest::testECEVPSaveLoadStreamNoPass()
 			assert (strPubE.str() == pubKey);
 			assert (key == key2);
 			assert (!(key != key2));
-			ECKey ecKeyNE("secp112r2");
+			ECKey ecKeyNE("secp256k1");
 			EVPPKey keyNE(&ecKeyNE);
 			assert (key != keyNE);
 			assert (!(key == keyNE));
@@ -448,7 +448,7 @@ void EVPTest::testECEVPSaveLoadFile()
 			assert (strPubE.str() == pubKey);
 			assert (key == key2);
 			assert (!(key != key2));
-			ECKey ecKeyNE("secp112r2");
+			ECKey ecKeyNE("secp256k1");
 			EVPPKey keyNE(&ecKeyNE);
 			assert (key != keyNE);
 			assert (!(key == keyNE));
-- 
2.13.6

