From a84744326fc00f7136545e81856d2ca445bbf1f7 Mon Sep 17 00:00:00 2001
From: Scott Talbert <swt@techie.net>
Date: Thu, 6 Feb 2020 15:14:56 -0500
Subject: [PATCH 1/2] Set SQLite thread safe mode when building unbundled

---
 Data/SQLite/CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Data/SQLite/CMakeLists.txt b/Data/SQLite/CMakeLists.txt
index fcce2e0d6..ee7d2a177 100644
--- a/Data/SQLite/CMakeLists.txt
+++ b/Data/SQLite/CMakeLists.txt
@@ -44,7 +44,10 @@ target_include_directories(DataSQLite
 
 if(POCO_UNBUNDLED)
         target_link_libraries(DataSQLite PUBLIC SQLite::SQLite3)
-	target_compile_definitions(DataSQLite PUBLIC POCO_UNBUNDLED)
+	target_compile_definitions(DataSQLite PUBLIC
+		POCO_UNBUNDLED
+		SQLITE_THREADSAFE=1
+	)
 else()
 	if(WINCE)
 		target_compile_definitions(DataSQLite PRIVATE SQLITE_MSVC_LOCALTIME_API)
-- 
2.24.1

