From 48cb7ba8a35dd658227824dd407ad0c31a318d0d Mon Sep 17 00:00:00 2001
From: Scott Talbert <swt@techie.net>
Date: Thu, 6 Feb 2020 15:17:57 -0500
Subject: [PATCH 2/2] Use ${LIB_SUFFIX} when setting PocoConfigPackageLocation

---
 CMakeLists.txt         | 2 +-
 cmake/PocoMacros.cmake | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 96c475828..d7ce6e512 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -443,7 +443,7 @@ write_basic_package_version_file(
 if (WIN32)
   set(PocoConfigPackageLocation "cmake")
 else()
-  set(PocoConfigPackageLocation "lib/cmake/${PROJECT_NAME}")
+  set(PocoConfigPackageLocation "lib${LIB_SUFFIX}/cmake/${PROJECT_NAME}")
 endif()
 
 configure_file(cmake/${PROJECT_NAME}Config.cmake.in "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}/${PROJECT_NAME}Config.cmake" @ONLY)
diff --git a/cmake/PocoMacros.cmake b/cmake/PocoMacros.cmake
index 14ebbb6c7..b99422dba 100644
--- a/cmake/PocoMacros.cmake
+++ b/cmake/PocoMacros.cmake
@@ -239,7 +239,7 @@ configure_file("cmake/Poco${target_name}Config.cmake"
 if (WIN32)
 	set(PocoConfigPackageLocation "cmake")
 else()
-	set(PocoConfigPackageLocation "lib/cmake/${PROJECT_NAME}")
+	set(PocoConfigPackageLocation "lib${LIB_SUFFIX}/cmake/${PROJECT_NAME}")
 endif()
 
 install(
-- 
2.24.1

