From 1a62d5f5c7bb2f29fadcac7e2a76c422c44eb258 Mon Sep 17 00:00:00 2001
From: Scott Talbert <swt@techie.net>
Date: Mon, 13 Nov 2017 21:46:07 -0500
Subject: [PATCH] Fix building XMLStreamParser with unbundled expat

---
 XML/include/Poco/XML/XMLStreamParser.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/XML/include/Poco/XML/XMLStreamParser.h b/XML/include/Poco/XML/XMLStreamParser.h
index 580a3e8f0..e10ae8a7e 100644
--- a/XML/include/Poco/XML/XMLStreamParser.h
+++ b/XML/include/Poco/XML/XMLStreamParser.h
@@ -30,7 +30,11 @@
 #include "Poco/XML/QName.h"
 #include "Poco/XML/ValueTraits.h"
 #include "Poco/XML/Content.h"
+#if defined(POCO_UNBUNDLED)
+#include <expat.h>
+#else
 #include <Poco/XML/expat.h>
+#endif
 #include <map>
 #include <vector>
 #include <string>
-- 
2.13.6

